app/publishing/publish.js

```diff
@@ -2,9 +2,9 @@ const retry = require('../retry')
 const { getFile } = require('../storage')
 const publishByEmail = require('./publish-by-email')
 
-const publish = async (email, filename) => {
+const publish = async (request, filename, personalisation) => {
   const file = await retry(() => getFile(filename))
-  return publishByEmail(email, file)
+  return publishByEmail(request, file, personalisation)
```

Comment:
so `request` here is `email` in `publish-by-email`?

doesn't match up in my mind

Reply:
My bad.  Originally my approach to this ticket was to pass the full request (inc scheme data) to the publish by email function and handle the personalisation mapping there.

However, that wouldn't work as the two higher level functions that call this have different data configurations.  So I changed the approach to handling personalisation before it gets this far.  However, forgot to change this name back.  

Fixed now.